src/policy/packages.h

```diff
@@ -89,8 +89,9 @@ bool IsChildWithParents(const Package& package);
  */
 bool IsChildWithParentsTree(const Package& package);
 
-/** Get the hash of these transactions' wtxids, concatenated in lexicographical order (treating the
- * wtxids as little endian encoded uint256, smallest to largest). */
+/** Get the hash of these transactions' wtxids, with wtxids treated as little
+ * endian numbers and sorted in increasing numeric order (i.e. with byte-reversed
+ * wtxid hashes sorted lexicographically.) */
```

Comment:
According to Wiki `little endian` is preferred with a dash: https://en.wikipedia.org/wiki/Endianness

But the part I find confusing is rather expressing 3 concepts in a single sentence - can we break it up a bit?

```suggestion
/** Get the hash of the wtxids of these transactions, treating each wtxid hash 
 * as a little-endian number and sorting them in ascending numeric order.
 * This is equivalent to reversing the bytes of each wtxid hash and sorting them lexicographically.
 */
```